src/common.cc

```diff
@@ -553,7 +553,9 @@ void print_cached(struct text_object *obj, char *p, int p_max_size)
 
 void print_evaluate(struct text_object *obj, char *p, int p_max_size)
 {
-	evaluate(obj->data.s, p, p_max_size);
+	std::vector<char> buf(max_user_text.get(*state));
```

Comment:
@brndnmtthws, @inzanity, having second thoughts here. Should it be `text_buffer_size` instead of `max_user_text`? :astonished: